Uplift/Views/DropdownView.swift

```diff
@@ -0,0 +1,190 @@
+//
+//  DropdownView.swift
+//  
+//
+//  Created by Cameron Hamidi on 10/7/19.
+//
+
+import Foundation
+import UIKit
+
+enum DropdownStatus {
+    case up, half, down
+}
```

Comment:
might be misunderstanding but wouldn't `open, half, closed` be better names?

Reply:
agree with kevin -- I know these were the cases used in the filter view controller, but I think more descriptive names would be better